src/Tizen.NUI/src/public/BaseComponents/Style/TextEditorStyle.cs

```diff
@@ -99,6 +99,12 @@ public class TextEditorStyle : ViewStyle
             propertyChanged: (bindable, oldValue, newValue) => ((TextEditorStyle)bindable).cursorWidth = (int?)newValue,
             defaultValueCreator: (bindable) => ((TextEditorStyle)bindable).cursorWidth);
 
+        /// <summary> The bindable property of GrabHandleColor. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static readonly BindableProperty GrabHandleColorProperty = BindableProperty.Create(nameof(GrabHandleColor), typeof(Color), typeof(TextEditorStyle), null,
```

Comment:
Is this public like TextField and TextField and TextLabel?

Reply:
This is internal because it is used for style inside.
but, in the case of TextLabelStyle/TextFieldStyle, the old style is maintained.
Afterwards, it seems necessary to change all BindableProperty to internal like TextEditorStyle.